inst/rmarkdown/templates/sim_article/skeleton/skeleton.Rmd

```diff
@@ -15,8 +15,12 @@ year: 2017
 abstract: "This paper describes the use of the \LaTeX `simauth.cls` class file for setting papers for Statistics in Medicine using Rmarkdown."
 acknowledgements: "We thank [Stack Overflow](https://stackoverflow.com/) for the invaluable support to our research."
 keywords: Class file; \LaTeX; Statist. Med.; Rmarkdown;
+header-includes: 
+- \usepackage{ctable}
```

Comment:
I see; I put it in the skeleton to provide the user with an example, I always find it easier to remove parts I do not want than digging up the right syntax for parts I may want to add at some point. Makes sense? If not, I will remove it and make another pull request.
Thanks @jjallaire!